en/full-node.md

```diff
@@ -453,7 +453,7 @@ followed by the argument `xzf` followed by the file name. The argument
 `xzf` means eXtract the gZipped tar archive File. For example, for a
 64-bit tar archive in your current directory, the command is:
 
-    tar xzf bitcoin-0.10.0-linux64.tar.gz
+    tar xzf bitcoin-0.11.0-linux64.tar.gz
 
 This will create the directory `bitcoin-0.10.0` within your current
```

Comment:
This string also needs to be updated.  Maybe search for other occurrences of "0.10" in the document?